azurerm/resource_arm_streamanalytics_job.go

```diff
@@ -943,6 +945,27 @@ func parseArmStreamAnalyticsOutputDatasource(data map[string]interface{}) (strea
 	}
 }
 
+func extractAndValidateRequiredProperty(props map[string]interface{}) func(string) (*string, error) {
+	return func(k string) (*string, error) {
+		value := props[k].(string)
+		if len(value) > 0 {
+			return utils.String(value), nil
+		}
+		return nil, fmt.Errorf("Missing value for required property `%s`", k)
```

Comment:
Should we report the name of the resource which is buggered.

Reply:
Yup - ok i'll make an update. I've just realised that this may not work. I thought all of the schema were strings but thats not the case so i'll have a look at this a little more